src/components/input/_autosuggest-macro.njk

```diff
@@ -21,9 +21,11 @@
         data-error-select="{{ params.errorMessageSelect }}"
         data-error-api="{{ params.errorMessageAPI }}"
         data-error-api-link-text="{{ params.errorMessageAPILinkText }}"
+        {% if params.options %}
```

Comment:
I think we also need to check that this is defined too

Reply:
Added.